src/doc/tutorials/MRI.rst

```diff
@@ -0,0 +1,284 @@
+.. _MRI:
+
+MRI
+===
+
+.. toctree::
+    :maxdepth: 2
+
+This tutorial provides a short introduction to visualizing MRI data using VisIt_. We'll be relying on the Analyze data format, which is developed at the Mayo Clinic.
+
+
+Open the dataset
+----------------
+
+This tutorial uses the `MRI <https://visit-dav.github.io/largedata/>`_ dataset.
```

Comment:
Would suggest using this link instead, https://visit-dav.github.io/largedata/datarchives/mri because that takes the user to the specific download page for that data which includes additional info if they are so inclined. So, would write like so...

This tutorial uses the `mri_tutorial_data <https://visit-dav.github.io/largedata/datarchives/mri>`_ dataset.

As an aside, would you mind handling in this PR similar changes to the data file links elsewhere in the tutorial manual pages such as here, https://github.com/visit-dav/visit/blob/develop/src/doc/tutorials/index.rst

Reply:
@markcmiller86 I'm not quite sure if I'll have time to update older links on this PR. I'm on my other project this week but am trying to quickly get these changes in for Friday. If I don't have time to update them here, I can certainly do it in a couple of weeks when I'm back on VisIt.

Reply:
No worries. Fixed in #4790